cmd/init.go

```diff
@@ -44,6 +45,10 @@ func NewInitCommand() cli.Command {
 				Value: 4096,
 				Usage: "Size (in bits) of RSA keypair to generate (example: 4096)",
 			},
+			cli.StringFlag{
+				Name:  "curve",
+				Usage: "Elliptic curve name. Must be one of P-224, P-256, P-384, P-521, or Ed25519.",
+			},
```

Comment:
Is there an enum or something whose values we could iterate through to get all these names? I can see this getting missed in a future update should we ever add support for a new curve.

Reply:
Good call, I created one.